src/Entities/Entity.h

```diff
@@ -300,6 +297,10 @@ class cEntity
 
 	// tolua_end
 
+	void SetHeight(double a_Height);
+
+	void SetWidth(double a_Width);
```

Comment:
Why exactly are these not exported anymore?

Reply:
In the APIDoc is written `Notes = "FIXME: Remove this from API",`

Reply:
It is okay to remove or should I revert?

Reply:
Merged as-is, we'll re-export these if they become needed.